### ISSUE TITLE
algo attribute unclear

### ISSUE THREAD

**Issue by mb38291** [volterraedge/terraform-provider-volterra#228]
The [resource documentation](https://registry.terraform.io/providers/volterraedge/volterra/latest/docs/resources/volterra_service_policy#spec-argument-reference) needs to be updated to include the possible values for `algo`: 

Based on https://docs.cloud.f5.com/docs/api/service-policy those are:

```
FIRST_MATCH
DENY_OVERRIDES
ALLOW_OVERRIDES
```

`FIRST_MATCH `appears to be the default.

**Comment by eissko** [comment]
I agree here, I've just implemented the service policy. I had to go to the API definition because from resource description it was not clear how to use this property.
